notification/app/controllers/Main.scala

```diff
@@ -0,0 +1,22 @@
+package controllers
+
+import javax.inject.Inject
+
+import play.api.Logger
+import play.api.libs.ws.WSClient
+import play.api.mvc.{Action, Controller}
+import services.NotificationConfiguration
+import scala.concurrent.ExecutionContext
+
+final class Main @Inject()(wsClient: WSClient,
+  msNotificationsConfiguration: NotificationConfiguration)(
+  implicit executionContext: ExecutionContext
+  ) extends Controller {
+
+
+  private val logger = Logger("main")
```

Comment:
I would either remove that unused variable, or use Logger(this.getClass) instead of a string. That would avoid having a logger copied/pasted with the wrong name.